### DISCUSSION TITLE
Feature Request/Suggestion: Documentation of Pits and Risers Locations

### DISCUSSION THREAD

**Discussion by teomcam** [netbox-community/devicetype-library#2051] [General]
I tried to post to ideas section but somehow form not going ahead.
I've been using NetBox to document and manage our network infrastructure and it's been an indispensable tool for our operations.. No tahnks enoigh to developers and sharing with the community. However, I've noticed that there seems to be a gap in its capabilities regarding the physical layer documentation, specifically for pits and risers.
In our network infrastructure, the accurate documentation of pits (underground access points) and risers (vertical cable conduits) is crucial for maintenance, troubleshooting, and planning future expansions. These elements are fundamental components of our infrastructure, especially in complex environments or large campus settings.
I'm reaching out to the community and developers to ask if there's an existing way to document this information in NetBox that I might have missed. If not, is there any interest or ongoing work to add this kind of functionality? Alternatively, could this be considered as a feature request for future releases?

**Comment by DanSheps** [comment]
This is not the apporpriate place to make this suggestion.  You would need to make it on the main netbox repository.
